src/modules/EdgeDetect/EdgeUtils.js

```diff
@@ -76,15 +76,15 @@ function sobelFilter(pixels, x, y) {
     }
   }
 
-  const grad = Math.sqrt(Math.pow(gradX, 2) + Math.pow(gradY, 2)),
-    angle = Math.atan2(gradY, gradX);
+  const grad = Math.sqrt(Math.pow(gradX, 2) + Math.pow(gradY, 2)), // gradient(magnitude)
+    angle = Math.atan2(gradY, gradX); // angle
```

Comment:
What is the purpose of this angle comment?

Reply:
Hmm. No purpose really. I think I should remove it.